data/geany.gtkrc

```diff
@@ -56,8 +56,8 @@ style "geany-compiler-error-style" {
     fg[ACTIVE] = "#ffff00000000"
 }
 style "geany-compiler-context-style" {
-    fg[NORMAL] = "#888800000000"
-    fg[ACTIVE] = "#888800000000"
+    fg[NORMAL] = "#7f7f00000000"
+    fg[ACTIVE] = "#7f7f00000000"
```

Comment:
I just went in python and did `hex(int((0x7f/255.0)*65536))` to scale from 8-bit to 16-bit. Would be better to use 8-bit HTML-style notation, if supported.